scripts/get-polyhaven-textures.py

```diff
@@ -66,14 +67,14 @@ def get_texture(name: str, res: str, odir: str, mdir: str) -> bool:
 		if 'disp' in textures:
 			r = subprocess.run(['vtex2', 'pack', '-n', '-q', '--normal-map', textures['nor_dx'], '--height-map', textures['disp'], '-o', f'{odir}/{name}_n.vtf'])
 		else:
-			r = subprocess.run(['vtex2', 'convert', '-q', '-f', 'rgba8888', '-o', f'{odir}/{name}_n.vtf', textures['nor_dx']])
+			r = subprocess.run(['vtex2', 'convert', '-q', '-f', 'bc7', '-o', f'{odir}/{name}_n.vtf', textures['nor_dx']])
```

Comment:
shouldn't normal maps be using ati1/2n?

Reply:
Don't you need some special magic in shaders for reading those?

Reply:
BC4 (ATI1N) is single channel, BC5 (ATI2N) is 2 channels. And yeah, ATI2N requires some shader magic to decode the normal out of that. BC7 will be much better quality.

Reply:
https://learn.microsoft.com/en-us/windows/win32/direct3d10/d3d10-graphics-programming-guide-resources-block-compression#compression-algorithms

![image](https://github.com/user-attachments/assets/7e0aa247-adb3-4c86-b90e-d0d731fda5be)

Reply:
btw this wasnt a blocking comment i was just curious, should have mentioned that

Reply:
> was just curious since i know strata fixed those up

Yeah we did, but it seems like the only shaders that still support ATI2N normals are LMG and water.
It'd be easy to add to PBR, but it'll add combos and isn't compatible with RGBA8888 normals that have heightmap data.